### DISCUSSION TITLE
How to disable some itens in Waybar?

### DISCUSSION THREAD

**Discussion by DouglasMartinEX** [prasanthrangan/hyprdots#1015] [Q&A]
Hey guys! I want to enable only clock, the workspaces and on/off pc button. How can i disable the other tabs like the network tab and system monitor?

**Comment by DouglasMartinEX** [comment]
edit: and how to change the windows background color to?

**Comment by kRHYME7** [comment]
https://github.com/prasanthrangan/hyprdots/wiki/Theming
Check this out

**Comment by prasanthrangan** [comment]
you can set your required modules in this file - `~/.config/waybar/config.ctl`

```
❯ cat ~/.config/waybar/config.ctl
1|28|bottom|( cpu memory ) ( clock )|wlr/workspaces hyprland/window|( network bluetooth pulseaudio custom/updates ) ( tray ) ( custom/wallchange custom/mode custom/wbar custom/cliphist custom/power )
0|0|top|( wlr/workspaces hyprland/window )|( clock )|( cpu memory ) ( network bluetooth pulseaudio custom/updates ) ( tray ) ( custom/wallchange custom/mode custom/wbar custom/cliphist custom/power )
0||bottom|[ custom/power custom/cliphist custom/wbar custom/mode custom/wallchange ] [ wlr/workspaces ]|[ clock ]|[ network bluetooth pulseaudio ] [ tray ]
```

where `|` is the delimiter and column,
 1. is 0 or 1, where 1 indicates the current mode in use
 2. is to set height of the bar (leave it empty <blank> to auto-scale for 3% of monitor res), the font size adapts to this height value.
 3. is postition of bar top/bottom
 4. is the left modules
 5. is the center modules
 6. is the right modules

**Comment by prasanthrangan** [comment]
> and how to change the windows background color to?

which window are you referring to??
